### PR TITLE
minor changes

### PR DESCRIPTION
**Pull request by NullFlex**
Untested, I haven't compiled this to see if this even works because I fucking hate java. 
this can't be fully cleaned up without a basically full recode, might do that in my spare time

### CODE REVIEW

**Comment by Techjar** [comment]
This seems visually identical and completely superfluous, and if performance was a concern it's beyond micro-optimization.

**Comment by NullFlex** [comment]
Disagree, it looks better on the eyes (although no performance changes)

File: src/org/vivecraft/command/ViveCommand.java [hunk 1]
```diff
@@ -44,20 +44,25 @@ public ViveCommand(VSE vse) {
...
 		}
 
 		String command = args[0].toLowerCase();
 
+
 		if (command.equals("vive-only") && sender.isOp()) {
-			if (args.length >= 2) {
-				if (args[1].toLowerCase().equals("true")) {
+			if (arg_length >= 2) {
+				//who wrote this garbage?
```
Comment (Techjar) [inline src/org/vivecraft/command/ViveCommand.java]: that's not nice :<

**Comment by NullFlex** [comment]
updated more stuff

**Pull request closed without merging**